sdk/requests/getTokenOrRefresh.js

```diff
@@ -1,19 +1,51 @@
 import { encode } from 'base-64';
 import { fetch } from 'react-native-ssl-pinning';
 import { Platform } from 'react-native';
-import { getParameters, setParameters } from '../configuration';
+import { getParameters, setParameters, eraseCode } from '../configuration';
 import { tokenEndpoint } from '../utils/endpoints';
-import REQUEST_TYPES from '../utils/constants';
+import { ERRORS, REQUEST_TYPES } from '../utils/constants';
+import { initializeErrors } from '../utils/helpers';
 
 const getTokenOrRefresh = async type => {
   const parameters = getParameters();
+  // En caso de que algún parámetro sea vacío, se rechaza la promise y se retorna el error correspondiente.
+  if (
+    !parameters.clientId ||
+    !parameters.redirectUri ||
+    !parameters.postLogoutRedirectUri ||
+    !parameters.clientSecret
+  ) {
+    const respError = initializeErrors(
+      parameters.clientId,
+      parameters.redirectUri,
+      parameters.postLogoutRedirectUri,
+      parameters.clientSecret,
+    );
+    // Se borra el parámetro code una vez ejecutado el getToken
+    eraseCode();
+    return Promise.reject(respError);
+  }
+
+  // En el caso de get token, se chequea que el code exista
+  if (type === REQUEST_TYPES.GET_TOKEN && !parameters.code) {
+    // Se borra el parámetro code una vez ejecutado el getToken
+    eraseCode();
+    return Promise.reject(ERRORS.INVALID_AUTHORIZATION_CODE);
+  }
+
+  // En el caso de refresh token, se chequea que el refresh token exista
+  if (type === REQUEST_TYPES.GET_REFRESH_TOKEN && !parameters.refreshToken){
```

Comment:
```suggestion
  if (type === REQUEST_TYPES.GET_REFRESH_TOKEN && !parameters.refreshToken) {
```
Para arreglar el linter